src/bors/command/parser.rs

```diff
@@ -127,10 +134,29 @@ fn parse_list<'a>(
     Some(Ok(result))
 }
 
+pub fn parse_key_value(input: &str) -> Result<HashMap<String, String>, String> {
```

Comment:
You can add `#[allow(unused)]` on top of the function for now to resolve the warnings, I'll use it later in another PR.

Reply:
Perfect